subsys/bluetooth/host/long_wq.c

```diff
@@ -0,0 +1,44 @@
+/* long_work.c - Workqueue intended for long-running operations. */
+
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+
+K_THREAD_STACK_DEFINE(bt_lw_stack_area, CONFIG_BT_LONG_WQ_STACK_SIZE);
+static struct k_work_q bt_long_wq;
+
+int bt_long_work_schedule(struct k_work_delayable *dwork, k_timeout_t timeout)
```

Comment:
I wasn't comfortable changing the api to long_wq since it implied some specific workqueue. `work` was more like "I have some long-running work I want to schedule, please deal with it".
But since I've since changed it to not 'degrade' to the syswq, then I've renamed it as you suggested